.gitignore

```diff
@@ -130,3 +130,5 @@ TEST.ipynb
 TrhCsCh.csv
```

Comment:
if some of these files are no longer in the package, we should delete the line from .gitignore. Please investigate 'TrhCsCh.csv', 'Untitled.ipynb' and 'tt.py'.